sycl/doc/extensions/experimental/sycl_ext_oneapi_work_group_static.asciidoc

```diff
@@ -157,88 +148,47 @@ SYCL 2020 requires that all global variables accessed by a device function are
 `const` or `constexpr`. This extension lifts that restriction for
 `work_group_static` variables.
 
-[NOTE]
-====
-Since `work_group_static` acts as a view, wrapping an underlying pointer, a
-developer may still choose to declare variables as `const`.
-====
-
-When `T` is a class type or bounded array, the size of the allocation is known
-at compile-time, and a SYCL implementation may embed the size of the allocation
-directly within a kernel. Each instance of `work_group_static<T>` is associated
+Each instance of `work_group_static<T>` is associated
 with a unique allocation in device local memory.
 
-When `T` is an unbounded array, the size of the allocation is unknown at
-compile-time, and must be communicated to the SYCL implementation via the
-`work_group_static_memory_size` property. Every instance of `work_group_static`
-for which `T` is an unbounded array is associated with a single, shared,
-allocation in device local memory. For example, two instances declared
-as `work_group_static<int[]>` and `work_group_static<float[]>` will be
-associated with the same shared allocation.
-
-If the total amount of device local memory requested (i.e., the sum of
-all memory requested by `local_accessor`, `group_local_memory`,
-`group_local_memory_for_overwrite` and `work_group_static`) exceeds a device's
-local memory capacity (as reported by `local_mem_size`) then the implementation
-must throw a synchronous `exception` with the `errc::memory_allocation` error
-code from the kernel invocation command (e.g. `parallel_for`).
-
 [source,c++]
 ----
-operator T&() const noexcept;
+operator T&() noexcept;
 ----
 _Returns_: A reference to the object stored in the device local memory
 associated with this instance of `work_group_static`.
 
 [source,c++]
 ----
-const work_group_static<T>& operator=(const T& value) const noexcept;
+work_group_static& operator=(const T& value) noexcept;
 ----
-_Constraints_: Available only if `std::is_array_v<T>>` is false.
+_Constraints_: Available only if `std::is_array_v<T>` is false.
 
 _Effects_: Replaces the value referenced by `*ptr` with `value`.
```

Comment:
There is no `ptr`.  Reword to avoid that reference.  Check for other occurrences of `ptr` in this document.

Reply:
I reword this, thanks